packages/transformers/html/src/HTMLTransformer.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import {Transformer} from '@parcel/plugin';
+
+import parse from 'posthtml-parser';
+import render from 'posthtml-render';
+import nullthrows from 'nullthrows';
+import collectDependencies from './dependencies';
+import extractInlineAssets from './inline';
+
+export default new Transformer({
+  canReuseAST(ast) {
+    return ast.type === 'posthtml' && ast.version === '0.0.0';
+  },
+
+  async parse(asset) {
+    return {
+      type: 'posthtml',
+      version: '0.0.0',
```

Comment:
probably should get a real version from posthtml-parser's package.json

Reply:
Yeah, I did this originally. It seems like posthtml doesn't really version anything (or break things significantly so far, from what I can tell). I'd be on board with this.